x-pack/plugins/cases/public/components/actions/tags/edit_tags_flyout.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useState } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutFooter,
+  EuiFlyoutHeader,
+  EuiLoadingSpinner,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import type { Case } from '../../../../common';
+import { useGetTags } from '../../../containers/use_get_tags';
+import { EditTagsSelectable } from './edit_tags_selectable';
+import * as i18n from './translations';
+import type { TagsSelectionState } from './types';
+
+interface Props {
+  selectedCases: Case[];
+  onClose: () => void;
+  onSaveTags: (args: TagsSelectionState) => void;
+}
+
+const EditTagsFlyoutComponent: React.FC<Props> = ({ selectedCases, onClose, onSaveTags }) => {
+  const { data: tags, isLoading } = useGetTags();
+
+  const [tagsSelection, setTagsSelection] = useState<TagsSelectionState>({
+    selectedTags: [],
+    unSelectedTags: [],
+  });
+
+  const onSave = useCallback(() => onSaveTags(tagsSelection), [onSaveTags, tagsSelection]);
+
+  const headerSubtitle =
+    selectedCases.length > 1 ? i18n.SELECTED_CASES(selectedCases.length) : selectedCases[0].title;
+
+  return (
+    <EuiFlyout
+      ownFocus
+      onClose={onClose}
+      aria-labelledby="cases-edit-tags-flyout"
+      data-test-subj="cases-edit-tags-flyout"
+      size="s"
+      paddingSize="m"
+    >
+      <EuiFlyoutHeader hasBorder>
+        <EuiTitle size="m">
+          <h2 data-test-subj="cases-edit-tags-flyout-title">{i18n.EDIT_TAGS}</h2>
+        </EuiTitle>
+        <EuiText color="subdued">
+          <p>{headerSubtitle}</p>
+        </EuiText>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        {isLoading ? (
+          <EuiLoadingSpinner />
+        ) : (
+          <EditTagsSelectable
```

Comment:
Can the selectable control be given more height - here I have about 15 tags, and there is a lot more height available in the flyout without the need to have the content scrollable:

<img width="530" alt="image" src="https://user-images.githubusercontent.com/7405507/197563754-4a1def97-8722-41a4-911b-4a80d1fefb07.png">

Reply:
Good point!

Reply:
Done!